pkg/ui/react-app/src/thanos/pages/blocks/BlockDetails.tsx

```diff
@@ -80,6 +82,39 @@ export const BlockDetails: FC<BlockDetailsProps & PathPrefixProps> = ({
             <b>Chunks:</b> <span>{block.stats.numChunks}</span>
           </div>
           <hr />
+          {sizeStats && (
+            <>
+              <div data-testid="total-size">
+                <b>Total size:</b>&nbsp;
+                <span title={sizeStats.totalBytes + ' Bytes'}>{humanizeBytes(sizeStats.totalBytes)}</span>
+              </div>
+              <div data-testid="chunk-size">
+                <b>Chunks:</b>&nbsp;
+                <span title={sizeStats.chunkBytes + ' Bytes'}>
+                  {humanizeBytes(sizeStats.chunkBytes)} ({((sizeStats.chunkBytes / sizeStats.totalBytes) * 100).toFixed(2)}%)
+                </span>
+              </div>
+              <div data-testid="index-size">
+                <b>Index:</b>&nbsp;
+                <span title={sizeStats.indexBytes + ' Bytes'}>
+                  {humanizeBytes(sizeStats.indexBytes)} ({((sizeStats.indexBytes / sizeStats.totalBytes) * 100).toFixed(2)}%)
+                </span>
+              </div>
+              <div data-testid="daily-bytes">
+                <b>Daily:</b>&nbsp;
```

Comment:
Not too happy about _Daily_ as label, but it's shorter than something like _Total per day_.